src/plugins/console/server/lib/spec_definitions/json/generated/snapshot.delete.json

```diff
@@ -9,6 +9,6 @@
     "patterns": [
       "_snapshot/{repository}/{snapshot}"
     ],
-    "documentation": "https://www.opensearch.org/guide/en/elasticsearch/reference/master/modules-snapshots.html"
+    "documentation": "https://opensearch.org/docs/latest/opensearch/snapshot-restore/"
```

Comment:
It doesn't look like any of these snapshot APIs are fully documented. I think it's okay to leave the link but add them to `noDocumentation`.